### DISCUSSION TITLE
Question about folders

### DISCUSSION THREAD

**Discussion by Technically-Possible** [photoprism/photoprism#4579] [General]
Is there any way to stop nested folders showing up? The way they currently show is every folder in every directory and the library is shown all at the same time. I've currently got it organized so I would like to have on the GUI the ability to click interfolders and see all the folders within the folder instead of every folder being displayed all at the same time

**Comment by graciousgrey** [comment]
A hierarchical folder view is planned but not available yet. At this time you can use the search to only see certain folders.
